content/logs/log_collection/fluentd.md

```diff
@@ -10,28 +10,60 @@ As long as you can forward your FluentD logs over tcp/udp to a specific port, yo
 
 In order to get the best use out of your logs in Datadog, it is important to have the proper metadata associated with your logs (including hostname and source). By default, the hostname and timestamp should be properly remapped thanks to our default [remapping for reserved attributes][2]. 
 
-### Source
+### Source and Custom tags
 
-Add the `ddsource` attribute in your logs in order to trigger the [integration automatic setup][3] in Datadog.
+Add the `ddsource` attribute with [the name of the log integration][6] in your logs in order to trigger the [integration automatic setup][3] in Datadog.
+[Host tags][5] are automatically set on your logs if there is a matching hostname in your [infrastructure list][4]. Use the `ddtags` attribute to add custom tags to your logs:
+
+Setup Example:
 
 ```
-{
-    "ddsource": "<MY_SOURCE_VALUE>"
-}
+# Match events tagged with "datadog.**" and
+# send them to Datadog
+<match datadog.**>
+
+  @type datadog
+  @id awesome_agent
+  api_key <your_api_key>
+
+  # Optional
+  include_tag_key true
+  tag_key 'tag'
+
+  # Optional tags
+  dd_source '<INTEGRATION_NAME>' 
+  dd_tags '<KEY1:VALUE1>,<KEY2:VALUE2>'
+  dd_sourcecategory '<SOURCE_CATEGORY>'
+
+</match>
 ```
 
-### Custom tags
+### Kubernetes and Docker tags
 
-[Host tags][5] are automatically set on your logs if there is a matching hostname in your [infrastructure list][4]. Use the `ddtags` attribute to add custom tags to your logs:
+Tags in Datadog are critical to be able to jump from one part of the product to the other. Having the right metadata associated to your logs is therefore important to jump from the container view or any container metrics to the most related logs.
```

Comment:
some grammar/syntax:
Datadog tags are critical to be able to jump from one part of the product to another. Having the right metadata associated with your logs is therefore important in jumping from a container view or any container metrics to the most related logs.

I also think there should be more explicit language around the "jumping" — is that referring to something in a UI view?